core/src/main/scala/filodb.core/memstore/PartKeyLuceneIndex.scala

```diff
@@ -660,7 +695,7 @@ class PartKeyLuceneIndex(ref: DatasetRef,
     }
     val collector = new ActionCollector(handleMatch)
     withNewSearcher(s => s.search(query, collector))
-    partIdFromPartKeyLookupLatency.record(Math.max(0, System.currentTimeMillis - startExecute))
+    partIdFromPartKeyLookupLatency.record(Math.max(0, System.nanoTime - startExecute))
```

Comment:
``Math.max`` can be avoided as nanoTime is monotonically increasing.

Reply:
done